src/command_modules/azure-cli-container/README.rst

```diff
@@ -47,6 +47,11 @@ Commands to create an Azure container group
         --registry-password           : The password to log in container image registry server.
         --registry-username           : The username to log in container image registry server.
 
+    Log Analytics Arguments
+        --log-analytics-workspace       : The Log Analytics workspace. Must be in the currently set subscription.
+        --log-analytics-workspace-id    : The Log Analytics workspace id.
```

Comment:
This does not conform with the command design guidelines:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters

`--log-analytics-workspace` should support name or ID.